### PR TITLE
chore(fab): Remove the mdc-fab--plain modifier

### LINKED ISSUE #1143: Update FAB color to primary color
- [x] Create color mixins
- [x] Call color mixin from stylesheet, with primary color
- [x] Update README to document color mixins
- [ ] Delete --plain modifier
- [ ] Include "breaking change" message about removing modifiers, point to color mixin as deprecation replacement.

### PR DESCRIPTION
**Pull request by lynnjepsen**
Resolves #1143, Update demo to use the mdc-fab-accessible mixin, since this is the migration path away from mdc-fab--plain.

BREAKING CHANGE: Removes mdc-fab--plain, please update your code to use mdc-fab-accessible mixin instead.

### CODE REVIEW

File: demos/fab.html [hunk 1]
```diff
@@ -93,52 +90,84 @@
...
+        <legend>FABs with Ripple</legend>
+        <div class="demo-fabs">
+          <figure>
+            <button class="mdc-fab material-icons" aria-label="Favorite">
+              <span class="mdc-fab__icon">
+                favorite_border
+              </span>
+            </button>
+            <figcaption>
+              <div>Themed FAM</div>
```
Comment (kfranqueiro) [inline demos/fab.html]: Typo: FAM -> FAB

(Insert "it's lit" meme here)

File: demos/fab.html [hunk 2]
```diff
@@ -93,52 +90,84 @@
...
+            </figcaption>
+          </figure>
+          <figure>
+            <button class="mdc-fab mdc-fab--mini material-icons" aria-label="Favorite">
+              <span class="mdc-fab__icon">
+                favorite_border
+              </span>
+            </button>
+            <figcaption>
```
Comment (kfranqueiro) [inline demos/fab.html]: Was your thought that including these captions on the first set of buttons is sufficient? (Just noting that they're only present on the ripple set, not the CSS-only set.)

File: demos/demos.scss [hunk 3]
```diff
@@ -23,6 +23,14 @@
 $mdc-theme-primary: #212121 !default; // Grey 900
 $mdc-theme-secondary: #64dd17 !default; // Light Green A700
 
+@import "../packages/mdc-theme/color_palette";
+@import "../packages/mdc-fab/mixins";
+
+
```
Comment (kfranqueiro) [inline demos/demos.scss]: Not sure this double-empty-line was intentional?

File: demos/fab.html (hunk 2 above)
Comment (lynnjepsen) [reply demos/fab.html]: Yes, I think it is sufficient. (And whenever I copy paste stuff across sections I run the risk of forgetting to update it in a later PR.)

Comment (kfranqueiro) [reply demos/fab.html]: Yup, can definitely agree with that logic and figured that might be part of the reason :)

**Comment by joshiste** [comment]
this wasn't updated in the components docs

**Comment by kfranqueiro** [comment]
Thanks @joshiste; the docs mirrored on material.io should be up to date now.

**Pull request merged**